ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/-types.ts

```diff
@@ -50,17 +54,33 @@ export const packageManagers = [
     id: 'Cargo',
     label: 'Cargo (Rust)',
   },
+  {
+    id: 'Carthage',
+    label: 'Carthage (Objective-C / Swift)',
+  },
+  {
+    id: 'CocoaPods',
+    label: 'CocoaPods (Objective-C / Swift)',
+  },
   {
     id: 'Composer',
     label: 'Composer (PHP)',
   },
+  {
+    id: 'Conan',
+    label: 'Conan 1.x (C / C++)',
+  },
   {
     id: 'GoMod',
     label: 'GoMod (Go)',
   },
+  {
+    id: 'Gradle',
+    label: 'Gradle Legacy (Java), mutually exclusive with Gradle',
+  },
   {
     id: 'GradleInspector',
-    label: 'Gradle (Java)',
+    label: 'Gradle (Java), mutually exclusive with Gradle Legacy',
```

Comment:
This is fine by me as is, but if you want to guard for Gradle and GradleInspector to not be enabled at the same time and learn some Zod while at it, it should be relatively simple with [Zod's `.refine()`](https://stackoverflow.com/a/75572374) in create-run-utils.ts for the `packageManagers` `z.object()`.

Reply:
I have applied the suggested change and verified that it works. Just one thing is a bit odd: When selecting both package managers the error is only shown after submitting the form. However, afterward the error message is updated just by clicking the checkboxes. So somehow it seems the validation is inactive as long as it was not tried to submit the form.

Reply:
Great! The form library we use [validates the input on submit by default](https://www.react-hook-form.com/api/useform/#mode), but validates already errored fields on change. It should probably be changed on `onBlur` to validate when an input is unfocused.

Reply:
> It should probably be changed on `onBlur` to validate when an input is unfocused.

Indeed that's generally how we should validate forms. Is this something we can easily change globally for all forms?

Reply:
I don't think it can be configured globally, but it's easy to do for each form by adding the `mode: 'onBlur'` parameter linked above to all `useForm` calls.